packages/wallet/src/mintbase-wallet.ts

```diff
@@ -173,9 +173,9 @@ export const MintbaseWallet: WalletBehaviourFactory<
 
     return account.signAndSendTransaction({
       receiverId: receiverId || contractId,
-      actions: actions.map((action) => createAction(action)) as nearAPI.transactions.Action[],
+      actions: actions.map((action) => createAction(action) as unknown as nearAPI.transactions.Action),
       walletCallbackUrl: callback,
-    });
+    })as unknown as FinalExecutionOutcome;
```

Comment:
And not this.

Reply:
Same change can be applied here. Note that these two completely different unsafe approaches to the exact same type cast.

Reply:
same here. easy to talk, difficult to fix. It should had been fixed on your near-api-js v3 p/r, it wasnt.

<img width="606" alt="Screenshot 2024-07-29 at 11 23 20" src="https://github.com/user-attachments/assets/63edee12-8c0d-4143-a694-c018f848f1be">